spec/paypal_express/remote/browser_helpers.rb

```diff
@@ -0,0 +1,41 @@
+require 'selenium-webdriver'
+
+module Killbill
+  module PaypalExpress
+    module BrowserHelpers
+      def login_and_confirm(url)
+        driver = Selenium::WebDriver.for :firefox
+        #login page
+        driver.get url
+        wait = Selenium::WebDriver::Wait.new(:timeout => 15)
+        driver.switch_to.frame('injectedUl')
+        email_element, pwd_element, login_element = wait.until {
+          email_element = driver.find_element(:id, 'email')
```

Comment:
I believe PayPal has two different versions of the UI. My (old) account still falls back to the legacy layout, for which the ids are:

* `login_email` (instead of `email`)
* `login_password` (instead of `password`)
* `submitLogin` (instead of `btnLogin`)
* `continue_abovefold` (instead of `confirmButtonTop`)

I would suggest looking for both elements each time, so we support both flows.